<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskFemConstraintRigidBody</class>
 <widget class="QWidget" name="TaskFemConstraintRigidBody">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lbl_info">
     <property name="text">
      <string>Select multiple face(s), click Add or Remove</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayout1">
     <item>
      <widget class="QToolButton" name="btnAdd">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemove">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Remove</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="lw_references"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lbl_ref_node">
       <property name="text">
        <string>Reference Node</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="f_layout_ref_node">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_ref_node_x">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="Gui::QuantitySpinBox" name="qsb_ref_node_x">
         <property name="singleStep">
          <double>1.0</double>
         </property>
         <property name="maximum">
          <double>1000000000.0</double>
         </property>
         <property name="unit" stdset="0">
          <string notr="true">mm</string>
         </property>
         <property name="value" stdset="0">
          <double>0.0</double>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_ref_node_y">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Gui::QuantitySpinBox" name="qsb_ref_node_y">
         <property name="singleStep">
          <double>1.0</double>
         </property>
         <property name="maximum">
          <double>1000000000.0</double>
         </property>
         <property name="unit" stdset="0">
          <string notr="true">mm</string>
         </property>
         <property name="value" stdset="0">
          <double>0.0</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_ref_node_z">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="Gui::QuantitySpinBox" name="qsb_ref_node_z">
         <property name="singleStep">
          <double>1.0</double>
         </property>
         <property name="maximum">
          <double>1000000000.0</double>
         </property>
         <property name="unit" stdset="0">
          <string notr="true">mm</string>
         </property>
         <property name="value" stdset="0">
          <double>0.0</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>


    <widget class="QGroupBox" name="gpb_trans_parameter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="title">
      <string>Translational Parameters</string>
     </property>
     <item>
      <layout class="QFormLayout" name="f_layout_trans">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_trans_x">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_trans_y">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_trans_z">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="Gui::QuantitySpinBox" name="qsb_force_x">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="singleStep">
          <double>1.00000000000000</double>
         </property>
         <property name="unit" stdset="0">
          <string notr="true">N<string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Gui::QuantitySpinBox" name="qsb_force_y">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="singleStep">
          <double>1.00000000000000</double>
         </property>
         <property name="unit" stdset="0">
          <string notr="true">N</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="Gui::QuantitySpinBox" name="qsb_force_z">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="singleStep">
          <double>1.00000000000000</double>
         </property>
         <property name="unit" stdset="0">
          <string notr="true">N</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="Gui::QuantitySpinBox" name="qsb_disp_x">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="singleStep">
          <double>1.00000000000000</double>
         </property>
         <property name="unit" stdset="0">
          <string notr="true">mm</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="Gui::QuantitySpinBox" name="qsb_disp_y">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="singleStep">
          <double>1.00000000000000</double>
         </property>
         <property name="unit" stdset="0">
          <string notr="true">mm</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="Gui::QuantitySpinBox" name="qsb_disp_z">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="singleStep">
          <double>1.00000000000000</double>
         </property>
         <property name="unit" stdset="0">
          <string notr="true">mm</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="cb_x_trans_mode"/>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="cb_y_trans_mode"/>
       </item>
       <item row="2" column="3">
        <widget class="QComboBox" name="cb_z_trans_mode"/>
       </item>
      </layout>
     </item>
    </widget>








    <widget class="QGroupBox" name="gpb_rot_parameter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="title">
      <string>Rotational Parameters</string>
     </property>



     <item>
      <layout class="QFormLayout" name="f_layout_rot">
       <item row="0" column="0">
        <widget class="QLabel" name="lbl_rot_x">
         <property name="text">
          <string>X:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lbl_rot_y">
         <property name="text">
          <string>Y:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lbl_rot_z">
         <property name="text">
          <string>Z:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="Gui::QuantitySpinBox" name="qsb_moment_x">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="singleStep">
          <double>1.00000000000000</double>
         </property>
         <property name="unit" stdset="0">
          <string notr="true">N*m</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="Gui::QuantitySpinBox" name="qsb_moment_y">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="singleStep">
          <double>1.00000000000000</double>
         </property>
         <property name="unit" stdset="0">
          <string notr="true">N*m</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="Gui::QuantitySpinBox" name="qsb_moment_z">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="singleStep">
          <double>1.00000000000000</double>
         </property>
         <property name="unit" stdset="0">
          <string notr="true">N*m</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="Gui::QuantitySpinBox" name="qsb_rot_x">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="singleStep">
          <double>1.00000000000000</double>
         </property>
         <property name="unit" stdset="0">
          <string notr="true">deg</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="Gui::QuantitySpinBox" name="qsb_rot_y">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="singleStep">
          <double>1.00000000000000</double>
         </property>
         <property name="unit" stdset="0">
          <string notr="true">deg</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="Gui::QuantitySpinBox" name="qsb_rot_z">
         <property name="keyboardTracking">
          <bool>false</bool>
         </property>
         <property name="singleStep">
          <double>1.00000000000000</double>
         </property>
         <property name="unit" stdset="0">
          <string notr="true">deg</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="cb_x_rot_mode"/>
       </item>
       <item row="1" column="3">
        <widget class="QComboBox" name="cb_y_rot_mode"/>
       </item>
       <item row="2" column="3">
        <widget class="QComboBox" name="cb_z_rot_mode"/>
       </item>
      </layout>
     </item>
    </widget>




    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Gui::QuantitySpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/QuantitySpinBox.h</header>
  </customwidget>
  <customwidget>
   <class>Gui::DoubleSpinBox</class>
   <extends>QWidget</extends>
   <header>Gui/SpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
